src/Validator/UserRequestValidator.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace App\Validator;
+
+use Codeception\Util\HttpCode;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpKernel\Exception\HttpException;
+use App\Repository\UsersRepository;
+
+class UserRequestValidator
+{
+
+    private $usersrepository;
+    public function __construct(UsersRepository $usersrepository)
+    {
+        $this->usersrepository = $usersrepository;
+    }
+
+    public function isJsonRequestTypeValid(Request $request): bool
+    {
+
+        if ($request->getContentType() == 'json') {
+            return true;
+        } else {
+            throw new HttpException(HttpCode::BAD_REQUEST, "Wrog data format");
+        }
+    }
+
+    public function isJsonParamsValid(array $params): bool
+    {
+        if (empty($params['name'])) {
+            throw new HttpException(HttpCode::BAD_REQUEST, "Name cannot be empty");
+        }
+        if (empty($params['email'])) {
+            throw new HttpException(HttpCode::BAD_REQUEST, "Email cannot be empty");
+        }
+        if (empty($params['password'])) {
+            throw new HttpException(HttpCode::BAD_REQUEST, "Password cannot be empty");
+        }
+
+        return true;
+    }
+
+    public function isPasswordValid(string $password): bool
```

Comment:
This function also could return void. Cuz you throw exception if something is wrong with data